bach/tests/functional/bach/test_df_database_create.py

```diff
@@ -0,0 +1,64 @@
+"""
+Copyright 2022 Objectiv B.V.
+"""
+from random import randrange
+
+import pytest
+
+from tests.functional.bach.test_data_and_utils import get_df_with_test_data, assert_equals_data
+from sql_models.sql_generator import to_sql
+from sql_models.util import quote_identifier
+
+
+def test_database_create_table(engine):
```

Comment:
For avoiding conflicts with parallelized tests. (who knows if there's another test reading `test_df_database_create__test_database_create_table` :woman_shrugging: )
```suggestion
@pytest.mark.xdist_group(name="db_writers")
def test_database_create_table(engine):
```

Reply:
Did change the table name slightly in d426795ff3bac2369d97d778913963dd0e26fcec, to be consistent with how we do this in test_df_sample.py. Removed a redundant `test`. So now it is `test_df_database_create__database_create_table`, before it was `test_df_database_create__test_database_create_table`

Reply:
then I think we should have a constant for for the x_dist group, like `XDIST_GROUP_FOR_DB_WRITE_TESTS = 'db_writer_group'` and use that constant for all  current xdist marked tests. I consider that a fast approach than making sure that all tests have unique table names

Reply:
We discussed this over video-chat. And I think the conclusion was that as it is this works. Not sure if we agreed that this was actually a better system than using `xdist_group`, but I do believe we both agreed that it works as is.

Sort of related discussion was this https://github.com/objectiv/objectiv-analytics/pull/976/files#r914069460